docs/_docs/interface.md

```diff
@@ -8,29 +8,28 @@ toc: false
 
 ## Home
 
-When you browse to the root of your registry, it's a fairly expected interface with
-links to collections and documentation. Here is what we see for the default "Tacosaurus Computing Center"
+When you browse to your registry's root, it's a reasonably expected interface with
+links to collections and documentation. Here is what we see for the default "Tacosaurus Computing Center".
 
 ![teams.png](../assets/img/interface/home.png)
 
-When you sign in with social authentication (e.g., GitHub) you are asked to sign standard terms of service.
-This basically says that the software is provided for you, free to use, and you are responsible
+When you sign in with social authentication (e.g., GitHub), you first must sign the standard terms of service (TOS).
+The TOS asserts that the software is provided for you, free to use, and you are responsible
 for taking care of your registry and are accountable for the containers within it.
 
 ![teams.png](../assets/img/interface/terms.png)
 
-When you agree, the robots welcome you!
+When you agree, robots welcome you!
```

Comment:
What is wrong with saying "the robots" ?

Reply:
It seemed unnecessary but now realize there are robots on the sregistry index URL. 

Reverted change.